packages/create-react-app/index.js

```diff
@@ -43,15 +43,15 @@ var chalk = require('chalk');
 var currentNodeVersion = process.versions.node;
 var semver = currentNodeVersion.split('.');
 var major = semver[0];
-var minor = semver[1];
+// var minor = semver[1];
 
-if (major < 4 || (major === 4 && minor < 7)) {
+if (major < 6) {
   console.error(
     chalk.red(
       'You are running Node ' +
         currentNodeVersion +
         '.\n' +
-        'Create React App requires Node 4.7 or higher. \n' +
+        'Create React App requires Node 6 or higher. \n' +
         'Please update your version of Node.'
     )
```

Comment:
Yes you're correct: https://github.com/facebookincubator/create-react-app/blob/master/packages/create-react-app/createReactApp.js#L400-L419.

What would you like the behavior to be on npm < 3?